src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -106,7 +122,7 @@ public bool TryRegister(SafeSocketHandle socket, out Interop.Error error)
         //
         private static readonly IntPtr MaxHandles = IntPtr.Size == 4 ? (IntPtr)int.MaxValue : (IntPtr)long.MaxValue;
 #endif
-        private static readonly IntPtr MinHandlesForAdditionalEngine = s_engineCount == 1 ? MaxHandles : (IntPtr)32;
+        private static readonly IntPtr MinHandlesForAdditionalEngine = s_maxEngineCount == 1 ? MaxHandles : (IntPtr)32;
```

Comment:
We could remove this.

And if it isn't removed, do we want to set this to `1` when the `DOTNET_SYSTEM_NET_SOCKETS_THREAD_COUNT` is set?

Reply:
Adam, you can ignore this. I had missed your reply https://github.com/dotnet/runtime/pull/35800#discussion_r421576408.